extensions-core/druid-ranger-security/pom.xml

```diff
@@ -193,7 +198,13 @@
             </exclusions>
             <scope>compile</scope>
         </dependency>
-
+        <!-- This is a transitive dependency of ranger-plugins-audit
+             added here to replace bloated bundle sdk -->
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-logs</artifactId>
+            <version>1.12.669</version>
```

Comment:
can we put a note next to the ranger plugin version to update the aws-sdk version when the ranger plugin gets updated?